javascript-practice/src/js/views/dashboardView.js

```diff
@@ -45,14 +45,14 @@ export default class DashboardView {
 
     // Error Add User Form Element
     this.addUserErrorEls = {
-      addFirstNameEl: document.querySelector('.add-first-name'),
-      addLastNameEl: document.querySelector('.add-last-name'),
-      addEmailIdEl: document.querySelector('.add-email-id'),
-      addMobileNoEl: document.querySelector('.add-mobile-no'),
-      addRoleEl: document.querySelector('.add-role'),
-      addUserNameEl: document.querySelector('.add-user-name'),
-      addPasswordEl: document.querySelector('.add-password'),
-      addConfirmPasswordEl: document.querySelector('.add-confirm-password'),
+      addFirstNameEl: document.querySelector('[data-error="firstNameError"]'),
```

Comment:
Should not use this solution. Based on input, you can find corresponding error element